Analysis

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+class Analysis:
```

Comment:
Could you add a multiline comment describing the class similar to what Asa and Carla did? It would help a lot documentation-wise since it would let us see a description of the class when we hover over it in an IDE.